kubectl-minio/cmd/tenant-create.go

```diff
@@ -85,6 +85,7 @@ func newTenantCreateCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 	f.StringVarP(&c.tenantOpts.NS, "namespace", "n", helpers.DefaultNamespace, "namespace scope for this request")
 	f.StringVarP(&c.tenantOpts.StorageClass, "storage-class", "s", helpers.DefaultStorageclass, "storage class for this MinIO tenant")
 	f.StringVarP(&c.tenantOpts.Image, "image", "i", helpers.DefaultTenantImage, "MinIO image for this tenant")
+	f.BoolVar(&c.tenantOpts.AntiAffinity, "anti-affinity", false, "enforce anti-affinity to ensure pods are not co-located replicas on a single node")
```

Comment:
Can we give it a different name? Something more intuitive, like:

--disable-host-sharing (or)
--scheduling-policy = default | oneReplicaPerNode

Reply:
changed it to `--disable-host-sharing`